src/lib/markers/implementations/TodayMarker.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const criticalStyles = {
+  position: 'absolute',
+  top: 0,
+  bottom: 0,
+  // FIXME: are these critical styles?
+  width: '2px',
+  backgroundColor: 'red'
+}
+
+// FIXME: this is used in all marker implementations
+// REVIEW: might want to memoize this as it creates a new object
+// in each render which is passed to React component
+const createMarkerStylesWithLeftOffset = leftOffset => ({
+  ...criticalStyles,
+  left: leftOffset
+})
+
+// eslint-disable-next-line
+const defaultRenderer = ({ styles }) => (
+  <div style={styles} data-testid="default-today-line" />
+)
+
+/** Marker that is placed based on current date.  This component updates itself on
+ * a set interval, dictated by the 'interval' prop.
+ */
+class TodayMarker extends React.Component {
```

Comment:
the logic here is very similar to `CustomMarker` and `CursorMarker`. I think maybe extend the `CutomMarkrt` to be the base component and extend its props to take every possible prop that `TodayMarker` and `CursorMarker` could take. This will save a lot of LOC and give the user even more ability to "replicate" the behavior of the other marker components

Reply:
Great point. Will look into that.